### PR TITLE
Follow up of #123 (Remove packaging of test data)

### PR DESCRIPTION
**Pull request by ericpre**
### Progress of the PR
- [x] Update base url,
- [x] Add custom pre-commit hook to update registry
- [n/a] update docstring (if appropriate),
- [x] update `CONTRIBUTING.rst`,
- [x] add an changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [x] Check formatting changelog entry in the `readthedocs` doc build of this PR (link in github checks)
- [n/a] add tests,
- [x] ready for review.

### CODE REVIEW

**Review by jlaehne — APPROVED** [review]

**Comment by ericpre** [comment]
@CSSFrancis: with the pre-commit hook.

Can someone cross checked that the pre-commit hook work as expected?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -60,14 +58,29 @@ To add or update test data:
       from rsciio.tests.registry_utils import update_registry
 
       update_registry()
-    
+  
+   On windows, you can use the :ref:`pre-commit hooks <pre-commit-hooks>` from the pull
```
Comment (jlaehne) [inline CONTRIBUTING.rst:62]: Here it says 'on windows', below it says 'unix only' - which one is right?

Comment (ericpre) [reply CONTRIBUTING.rst]: I tweaked the wording to make it less ambiguous. On windows, it is possible to use the pre-commit hook through pre-commit.ci to run on the pull request by requesting it in a comment (on the pull request).

**Comment by ericpre** [comment]
@jlaehne, do you want to have another review for this PR, or it is good to merge?

File: rsciio/tests/registry_utils.py [hunk 2]
```diff
@@ -36,7 +35,8 @@ def update_registry():
     ending.
     """
     if sys.platform == "win32":
-        raise RuntimeError("This is not supported on Windows.")
+        warnings.warn("This is not supported on Windows. Nothing done.")
```
Comment (jlaehne) [inline rsciio/tests/registry_utils.py:38]:
```suggestion
        warnings.warn("Updating registry is not supported on Windows. Nothing done.")
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -60,14 +58,29 @@ To add or update test data:
...
 
 As quality assurance, to improve the code, and to ensure a generalized
 functionality, pull requests need to be thoroughly reviewed by at least one
 other member of the development team before being merged.
 
+.. _pre-commit-hooks:
+
+Pre-commit Hooks
+----------------
+Two pre-commit hooks are setup:
```
Comment (jlaehne) [inline CONTRIBUTING.rst:76]:
```suggestion
Two pre-commit hooks are set up:
```

**Review by jlaehne — APPROVED** [review]

**Comment by jlaehne** [comment]
> @jlaehne, do you want to have another review for this PR, or it is good to merge?

I added two small phrasing corrections. Otherwise, looks good to me - though I have not run a pre-commit hook locally yet.

**Comment by ericpre** [comment]
Thanks @jlaehne: I tried it on linux and in https://github.com/hyperspy/rosettasciio/pull/131#issuecomment-1572655610 and it works well!

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -20,10 +20,11 @@ Pull Requests
 =============
 
 If you want to contribute to the RosettaSciIO source code, you can send us a
-`pull request <https://github.com/hyperspy/rosettasciio/pulls>`_. Small bug fixes
-are corrections to the user guide are typically a good starting point. But don't
-hesitate also for significant code contributions, such as support for a new
-file format - if needed, we'll help you to get the code ready to common standards.
+`pull request <https://github.com/hyperspy/rosettasciio/pulls>`_ against the ``main``
```
Comment (ericpre) [inline CONTRIBUTING.rst:23]: While I was around there: I specified "against the `main` branch" to clarify https://github.com/hyperspy/rosettasciio/issues/97#issuecomment-1573518005.

**Pull request merged**